1skeleton/7.0/overload.gradle

```diff
@@ -3,8 +3,9 @@ ext {
 			version: [
 					major: 7,
 					minor: 0,
-					rev: 0
+					rev: 1,
+					label: 2
```

Comment:
This skeleton does not need to be changed. A skeleton only contains the supporting files (which remain the same for a release series), so there is no need to add a specific version for them.

Reply:
Thanks for the explanation. At what point _does_ it need a change then? Major versions?